polynote-kernel/src/main/scala/polynote/kernel/lang/LanguageKernel.scala

```diff
@@ -2,60 +2,48 @@ package polynote.kernel.lang
 
 import java.io.File
 
-import fs2.concurrent.{Enqueue, Queue}
+import fs2.Stream
 import polynote.kernel._
-import polynote.kernel.util.{Publish, RuntimeSymbolTable, SymbolDecl}
+import polynote.kernel.context.{RuntimeContext, SymbolDecl}
+import polynote.kernel.util.{Publish, RuntimeSymbolTable}
 
 /**
   * The LanguageKernel runs code in a given language.
   */
 trait LanguageKernel[F[_]] {
 
   // LanguageKernel is expected to have a reference to the shared runtime symbol table of a notebook
-  val symbolTable: RuntimeSymbolTable
+  val runtimeContext: RuntimeContext
```

Comment:
Can this be avoided?

Reply:
I definitely want to get rid of it, I needed it because of the path-dependent types in the other methods :(

Reply:
Got rid of it :)